staging/src/k8s.io/kubectl/pkg/cmd/get/get.go

```diff
@@ -454,17 +460,16 @@ func (o *GetOptions) transformRequests(req *rest.Request) {
 }
 
 // Run performs the get operation.
-// TODO: remove the need to pass these arguments, like other commands.
-func (o *GetOptions) Run(f cmdutil.Factory, cmd *cobra.Command, args []string) error {
+func (o *GetOptions) Run() error {
```

Comment:
Thanks for the PR. However `completion` command relies on `get` command's run function. Before fixing the dependency in completion, unfortunately we can not do that change.

Reply:
Thanks for the review. I looked at the relevant code in `util/completion` and it's hard to fix. I'd rather leave it as it is. I will close the PR.